ddl/util/syncer.go

```diff
@@ -58,8 +62,8 @@ var (
 	// SyncerSessionTTL is the etcd session's TTL in seconds.
 	// and it's an exported variable for testing.
 	SyncerSessionTTL = 90
-	// WaitTimeWhenErrorOccured is waiting interval when processing DDL jobs encounter errors.
-	WaitTimeWhenErrorOccured = 1 * time.Second
+	// ddlLogCtx uses for log.
+	ddlLogCtx = context.Background()
```

Comment:
Lots of `logutil.Logger(context.Background())` code snippets exist in this repo, I think we can provide a utility function `logutil.BgLogger()` to improve it.

Reply:
We can refactor it in other PR.

Reply:
OK